lapce-ui/src/problem.rs

```diff
@@ -302,6 +327,11 @@ impl Widget<LapceTabData> for ProblemContent {
                 ),
             );
 
+            if *data.problem.fold.get(path).unwrap_or(&false) {
```

Comment:
Maybe we should move this down the parent folder paint logic so that we still paint the parent folder.